frontend/src/index.tsx

```diff
@@ -1,14 +1,31 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import { createGlobalStyle } from 'styled-components';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
+const GlobalStyle = createGlobalStyle`
+  body {
+    margin: 0;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+      'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+      sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+  }
+  
+  code {
+    font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+      monospace;
+  }
+`;
+
 ReactDOM.render(
   <React.StrictMode>
+    <GlobalStyle />
     <App />
```

Comment:
I know this works correctly, I just have a question on how this works: Should you be editing the **App** style rather than adding a new HTML element with the appropriate font settings?

Reply:
These are global styles (i.e. things applied on tags such as 'body' or 'html') meaning I don't think it will work if you apply the style on App (a div). The current code comes from [this post](https://stackoverflow.com/questions/46760861/styled-components-how-to-set-styles-on-html-or-body-tag).